canvas/base.go

```diff
@@ -3,7 +3,6 @@ package canvas // import "fyne.io/fyne/canvas"
 
 import (
 	"sync"
-
```

Comment:
`goimports` wants this line to remain. I would recommend using our pre-commit git hook so you don't accidentally commit things that break the build. https://github.com/fyne-io/fyne/wiki/GitHooks